LibGUI/GWindow.h

```diff
@@ -159,4 +166,8 @@ class GWindow : public CObject {
     bool m_resizable { true };
     bool m_fullscreen { false };
     bool m_show_titlebar { true };
+    bool m_keybind_mode { false };
+    String m_entered_keybind;
+    size_t m_max_keybind_length;
```

Comment:
Let's always initialize primitive type members with something, just as a good habit:
```suggestion
    size_t m_max_keybind_length { 0 };
```